src/com/sittinglittleduck/DirBuster/Worker.java

```diff
@@ -405,7 +217,7 @@ else if(work.getBaseCaseObj().isUseRegexInstead())
                                 System.out.println("DEBUG Worker[" + threadId + "]: Found Dir (regex) " + url.toString());
```

Comment:
I know this is existing code, but be good to have this check Config.debug as well, or use log4j ;)